include/vcpkg/base/downloads.h

```diff
@@ -34,6 +35,10 @@ namespace vcpkg
     std::vector<int> download_files(const Filesystem& fs,
                                     View<std::pair<std::string, Path>> url_pairs,
                                     View<std::string> headers);
+
+    bool send_snapshot_to_api(const std::string& github_token,
```

Comment:
```suggestion
    bool send_dependency_snapshot_to_github(const std::string& github_token,
```
Hm I don't know if this should belong to this header. It is not really a function that can be reused by other part of vcpkg-tool.

Reply:
Good point, I will make the naming change in a separate PR. This function is used by `set-installed`, and see the earlier comment on why the function was moved into `downloads`